src/main/config/secor.common.properties

```diff
@@ -20,6 +20,8 @@
 # Regular expression matching names of consumed topics.
 secor.kafka.topic_filter=.*
 secor.kafka.topic_blacklist=
+# Adding files on S3 path with custom topic name 
+# Example: secor.kafka.customtopicname.topicname1=customtopicname1
```

Comment:
Maybe this shouldnt' be at the must set session

Reply:
Thanks for suggestion. Do you have any alternative? I think this place are important to set this configuration because the set of topics are defined here (secor.kafka.topic_filter). It's more cohesive to associate a map of custom names.

Reply:
based on the conventions of other parameters, this probably should be named: secor.kafka.custom_topic_name.  And these shouldn't be mandatory parameters.  Probably doesn't need to be specified here.